v4/export/prepare.go

```diff
@@ -23,19 +23,63 @@ func prepareDumpingDatabases(conf *Config, db *sql.DB) ([]string, error) {
 	}
 }
 
-func listAllTables(db *sql.DB, databaseNames []string) (DatabaseTables, error) {
+func listAllTables(db *sql.DB, databaseNames []string, skipViews bool) (DatabaseTables, error) {
```

Comment:
why not providing a `lisAllViews` function?

Reply:
If we use `listAllViews()`, there is another step to calculate the difference between two maps if `no-views` is specified, which is inefficiency.